libcontainer/cgroups/devices/systemd.go

```diff
@@ -239,3 +239,43 @@ func allowAllDevices() []systemdDbus.Property {
 		newProp("DeviceAllow", []deviceAllowEntry{}),
 	}
 }
+
+func getCharEntryPath(rule *devices.Rule) string {
+	if isNVIDIADevice(rule) {
+		return getNVIDIAEntryPath(rule)
+	}
+	return fmt.Sprintf("/dev/char/%d:%d", rule.Major, rule.Minor)
+}
+
+func isNVIDIADevice(rule *devices.Rule) bool {
+	// NVIDIA device has major 195 and 507
+	if rule.Major == 195 || rule.Major == 507 {
+		return true
+	}
+	return false
+}
+
+func getNVIDIAEntryPath(rule *devices.Rule) string {
+	path := "/dev"
+	switch rule.Major {
+	case 195:
+		switch rule.Minor {
+		case 254:
+			path = fmt.Sprintf("%s/%s", path, "nvidia-modeset")
```

Comment:
I think these can just return the string; using the `Sprintf` with a fixed (`/dev`) variable makes the code harder to read, and may give the impression that the `path` variable may be manipulated further down.